extra/twig-extra-bundle/src/TwigExtraBundle.php

```diff
@@ -11,10 +11,19 @@
 
 namespace Twig\Extra\TwigExtraBundle;
 
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Twig\Extra\TwigExtraBundle\DependencyInjection\Compiler\MissingExtensionSuggestorPass;
 
 class TwigExtraBundle extends Bundle
 {
+    public function build(ContainerBuilder $container)
+    {
+        parent::build($container);
+
+        $container->addCompilerPass(new MissingExtensionSuggestorPass());
+    }
+
     public function getPath()
     {
         return \dirname(__DIR__);
```

Comment:
if `src/Resources/config` this should be removed, relying on the default (`src/`).